translate/cloud-client/translate_v3_create_and_delete_glossary_test.py

```diff
@@ -0,0 +1,49 @@
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+import pytest
+import translate_v3_create_glossary
+import translate_v3_delete_glossary
+import uuid
+
+PROJECT_ID = os.environ['GCLOUD_PROJECT']
+MODEL_ID = os.environ['AUTOML_TRANSLATION_MODEL_ID']
```

Comment:
remove this MODEL_ID

Reply:
Once this is updated to a MODEL_ID, the tests should all pass.